lib/wallet/rpc.js

```diff
@@ -899,8 +891,7 @@ class RPC extends RPCBase {
   }
 
   async listAddressGroupings(args, help) {
-    if (help)
-      throw new RPCError(errs.MISC_ERROR, 'listaddressgroupings');
+    // Not implemented.
```

Comment:
nit: extraneous comment (repeats the below)

Reply:
Just trying to be consistent with the style :-)
https://github.com/bcoin-org/bcoin/blob/master/lib/wallet/rpc.js#L1327